### ISSUE TITLE
A color by Tool Category

### ISSUE THREAD

**Issue by simonaubertbd** [amphi-ai/amphi-etl#161]
Hello,

The tools quantity and the tool categories quantities increased (which is pretty cool by the way^^).

However, having a different tool color by category would really help to understand a complex workflow. This is also a common norm in visual/data preparation tools

Best regards,

Simon

**Comment by tgourdel** [comment]
Yeah that's becoming hard to find the right tools indeed. Need to find the right approach. Was thinking using colors for something else. Will think about it :)
Thanks Simon!!

**Comment by simonaubertbd** [comment]
@tgourdel The advantage of using color for that is that it's pretty common and really easy to use (probably also easy to develop).
What had you in mind for color ?

Best regards,

Simon

**Comment by tgourdel** [comment]
Wanted to use colors to show where the components is ran when it's "in-database".

**Comment by simonaubertbd** [comment]
@tgourdel Ok. Maybe more a border color for that  ?
